package.json

```diff
@@ -103,5 +103,8 @@
     "url-parse": "^1.5.0",
     "y18n": "^4.0.1"
   },
-  "dependencies": {}
+  "dependencies": {
+    "@fluentui/react": "^8.75.1",
+    "@fluentui/react-northstar": "^0.63.1"
```

Comment:
Probably we shouldn't use northstar and "normal" react. We should just pick one flavor of react and use that (and I would vote for "not northstar")